crates/iceberg/src/error.rs

```diff
@@ -28,6 +28,9 @@ pub type Result<T> = std::result::Result<T, Error>;
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 #[non_exhaustive]
 pub enum ErrorKind {
+    /// The operation was rejected because the system is not in a state required for the operation’s execution.
```

Comment:
The description from error status is borrowed from https://grpc.io/docs/guides/status-codes/#the-full-list-of-status-codes